packages/react-components/react-combobox/docs/Spec.md

```diff
@@ -22,20 +24,23 @@ Select provides better mobile support and accessibility than Combobox, and has t
 
 Use Select when a basic single-select form component with no freeform text input or filtering is needed.
 
-### When to use Combobox
+### When to use Combobox or Dropdown
 
-Combobox is a more feature-rich version of Select, which comes at the cost a larger code footprint, and less robust support for accessibility compared to the native `<select>` element.
+Dropdown is a more feature-rich version of Select, which comes at the cost a larger code footprint, and less robust support for accessibility compared to the native `<select>` element. Combobox (as mentioned previously) is essentially a Dropdown that allows text input.
 
-Use Combobox when any of the following are required:
+Use Dropdown over Select when any of the following are required:
 
-- Filtering or freeform text input
 - Virtualization
 - Control over styling the dropdown and options
 - Multiple selection
 
+Use Combobox when filtering or freeform text input is needed.
```

Comment:
```suggestion
Use Combobox when you need any of Dropdown's features plus 
- filtering 
- freeform text input
```

Reply:
Tweaked the wording slightly (actually had already changed it based on one of your earlier comments 😅). Let me know if that looks good!